libc/test/include/CMakeLists.txt

```diff
@@ -17,7 +17,18 @@ add_libc_test(
 
 # stdbit_test only tests our generated stdbit.h, which is not generated in
 # overlay mode.
-if(LLVM_LIBC_FULL_BUILD AND libc.include.stdbit IN_LIST TARGET_PUBLIC_HEADERS)
+message(STATUS "TARGET_PUBLIC_HEADERS is ${TARGET_PUBLIC_HEADERS}")
+if(LLVM_LIBC_FULL_BUILD AND libc.include.stdbit IN_LIST TARGET_PUBLIC_HEADERS
+    AND libc.src.stdbit.stdc_leading_zeros_uc IN_LIST TARGET_LLVMLIBC_ENTRYPOINTS
```

Comment:
you don't need to check for all of these in the entrypoints list. If you just add them to the dependency list it will skip the test if they aren't available. The only one that currently needs to be checked manually is the header.

Reply:
So I think the issue with that approach though would be that those then get linked into the test?

Hmm...what do you think about this approach instead?
https://github.com/llvm/llvm-project/pull/80323